frontend/src/metabase/services.js

```diff
@@ -613,3 +613,9 @@ export const ApiKeysApi = {
   edit: PUT("/api/api-key/:id"),
   regenerate: PUT("/api/api-key/:id/regenerate"),
 };
+
+export const CacheConfigApi = {
+  list: GET("/api/cache"),
+  update: PUT("/api/cache"),
+  delete: DELETE("/api/cache"),
+};
```

Comment:
Sorry, I think I already called this out but to be sure, can we have a follow up task to move this to RTKQuery?

Reply:
Def, I've got a follow-up task in Milestone 2 for this